packages/bun-types/bun.d.ts

```diff
@@ -3753,6 +3753,43 @@ declare module "bun" {
    */
   function sleepSync(ms: number): void;
 
+  /**
+   * Parse a time string and return milliseconds, or format milliseconds as a string.
+   *
+   * Drop-in replacement for the `ms` npm package with compile-time inlining support.
+   *
+   * @example
+   * ```ts
+   * Bun.ms("2d")      // 172800000
+   * Bun.ms("1.5h")    // 5400000
+   * Bun.ms("1m")      // 60000
+   * Bun.ms("5s")      // 5000
+   * Bun.ms("100ms")   // 100
+   * Bun.ms("1mo")     // 2629800000
+   * Bun.ms("1y")      // 31557600000
+   * Bun.ms("100")     // 100
+   * Bun.ms(60000)     // "1m"
+   * Bun.ms(60000, { long: true })  // "1 minute"
+   * Bun.ms("invalid") // NaN
+   * ```
+   *
+   * Supports these units:
+   * - `ms`, `millisecond`, `milliseconds`
+   * - `s`, `sec`, `second`, `seconds`
+   * - `m`, `min`, `minute`, `minutes`
+   * - `h`, `hr`, `hour`, `hours`
+   * - `d`, `day`, `days`
+   * - `w`, `week`, `weeks`
+   * - `mo`, `month`, `months`
+   * - `y`, `yr`, `year`, `years`
+   *
+   * @param value - Time string to parse or milliseconds to format
+   * @param options - Formatting options (when value is a number)
```

Comment:
This is a pointless comment -- what do the options enable me to do.

```suggestion
   * @param options - options.long outputs the long-hand string representation.
```

You obviously then need to describe what the long-hand string representation is in the main body of the docblock.

That being said, I would actually declare the `options` type as its own thing:

```ts
interface MsNumericFormattingOptions {
    /**
     * Enable long-hand formatting.
     *
     * Long-hand formatting outputs values with the full unit name.
     * For example: `3 milliseconds` instead of `3ms`.
     */
    long?: boolean;
}
```